test.py

```diff
@@ -28,6 +28,7 @@ def sample_recarray():
 class SomeSchema(Schema):
     x = Array(dtype=np.float)
     y = Array()
+    z = ArrayOrNone()
```

Comment:
As we're adding more things to test, it is probably a good idea to name these things in a more readable way.